prime-router/src/main/kotlin/azure/observability/event/AzureEventUtils.kt

```diff
@@ -21,4 +21,17 @@ object AzureEventUtils {
             ObservationSummary(conditions)
         }
     }
+
+    /**
+     * For tracking the Message ID, we need the bundle.identifier value and system
+     * In FHIR bundle.identifier is optional so one or both may not be present
+     */
+    data class MessageID(val value: String? = null, val system: String? = null)
+
+    /**
+     * Returns the bundle identifier elements that relate to MSH-10 for tracking
+     */
+    fun getIdentifier(bundle: Bundle): MessageID {
```

Comment:
nit: Should logic to perform operations on FHIR bundles be moved outside of AzureEventUtils? We've already established this pattern here so that's why this is a "nit".

Reply:
I did go back and forth on this a little, for the moment I do like it here with the other Azure bundle operation. Just to note for the future: I could really only find one other use of the bundle identifier ([here](https://github.com/CDCgov/prime-reportstream/blob/095f3ecd8e8a1265abd63560979e0172474b0ee8/prime-router/src/main/kotlin/fhirengine/engine/ProcessedItem.kt#L31-L33)). I think if bundle.identifier gets used again, moving all of this to FHIRBundleHelpers.kt becomes more necessary.